.circleci/config.yml

```diff
@@ -19,6 +19,10 @@ node-core-base: &node-core-base
       run:
         name: Install dependencies
         command: yarn install --ignore-engines
+    - &node-gyp-install
+      run:
+        name: Install node-gyp
+        command: yarn global add node-gyp --ignore-engines
```

Comment:
This should be added to the devDependencies instead.

Reply:
I tried that first. It seems when `node-gyp` isn't installed globally, it fails to create `~/.cache/node-gyp` and then a bunch of cascading failures happen from that.

Reply:
Why did it work before? Is there something missing in `bindings.gyp` in order to switch to N-API?

Reply:
What does "before" mean? The CI didn't ever build the native addon until now.

Reply:
It did in several jobs.